### PR TITLE
Code coverage: User event service - 100%

### PR DESCRIPTION
**Pull request by Dimple16**
Screenshot 2023-02-20 at 10 06 55 PM

### CODE REVIEW

File: src/app/core/services/user-event.service.spec.ts [hunk 1]
```diff
@@ -2,15 +2,57 @@ import { TestBed } from '@angular/core/testing';
...
+
+  it('should emit internalSubject when calling internalError', () => {
+    const data = { message: 'Internal error' };
+    service.onInternalError(callback);
+    service.internalError(data);
+    expect(callback).toHaveBeenCalledOnceWith(data);
+  });
+
+  it('should emit outdatedClientSubject when calling outdatedClientError', () => {
+    const data = { message: 'Outdated client error' };
```
Comment (rvab) [inline src/app/core/services/user-event.service.spec.ts:47]: is this still valid?

File: src/app/core/services/user-event.service.spec.ts [hunk 2]
```diff
@@ -2,15 +2,57 @@ import { TestBed } from '@angular/core/testing';
 
 import { UserEventService } from './user-event.service';
 
-xdescribe('UserEventService', () => {
+describe('UserEventService', () => {
   let service: UserEventService;
```
Comment (rvab) [inline src/app/core/services/user-event.service.spec.ts:6]: `service` -> `userEventService`?

**Review by rvab — APPROVED** [review]
Minor comments

File: src/app/core/services/user-event.service.spec.ts (hunk 2 above)
Comment (Dimple16) [reply src/app/core/services/user-event.service.spec.ts]: fixed

**Pull request merged**